README.md

```diff
@@ -30,6 +30,8 @@ You will also need to download [spaCy's](https://spacy.io/models/en) `en_core_we
 python -m spacy download en_core_web_sm
 ```
 
+Note that this package was developed on MacOS and tested on Ubuntu, as such it may not work smoothly on Windows.
+
```

Comment:
```suggestion
Note that this package was developed on MacOS and tested on Ubuntu. Changes have been made to be compatible on a Windows system but are not tested and cannot be guaranteed. 
```
I guess because you've made those changes it should work more smoothly?